server/game/cards/locations/01/castleblack.js

```diff
@@ -1,15 +1,16 @@
 const DrawCard = require('../../../drawcard.js');
 
 class CastleBlack extends DrawCard {
-    setupCardAbilities() {
+    setupCardAbilities(ability) {
         this.action({
             title: 'Kneel this card to stand and give a defending character +2 STR',
+            cost: ability.costs.kneelSelf(),
             method: 'kneel'
         });
     }
 
     kneel(player) {
-        if(this.location !== 'play area' || !this.game.currentChallenge || this.kneeled) {
+        if(this.location !== 'play area' || !this.game.currentChallenge) {
```

Comment:
thanks @ystros ! all instances adapted as you suggested, and i'll take care of this in future porting work too